presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -987,6 +991,12 @@ std::string PrestoServer::getBaseSpillDirectory() const {
   return SystemConfig::instance()->spillerSpillPath().value_or("");
 }
 
+void PrestoServer::enableRuntimeStatsReporting() {
```

Comment:
```
s/enableRuntimeStatsReporting/enableRuntimeMetricReporting/
```
This is metric but not stats. Thanks!

Reply:
The base class is called BaseStatsReporter and we have defined StatsType enum as well. We seem to use the metric/stata interchangeably throughout.

Reply:
RuntimeStats means the ad hoc added runtime stats. Metric is different: https://facebookincubator.github.io/velox/metrics.html. We should rename BaseStatsReporter to BaseMetricReporter.